.github/workflows/build-images-ci.yaml

```diff
@@ -10,13 +10,15 @@ on:
   push:
     branches:
       - master
+      - ft/master/**
```

Comment:
IIRC, forked repo is recommended, so I reckon this feature branch will be created before hand by maintainer :thinking:

Reply:
Indeed, that's what we have in the CFP as well AFAIK.

Reply:
Yep, that should be acceptable as I think feature branches will be "stable" (I'm not expecting them to get regularly created or destroyed)

Reply:
Why is the same not needed for e.g., `ft/v1.10/**`? And if the reason is that feature branches are only expected against master, then why do we need the `master/` prefix?

Reply:
I expect the `ft/v1.10/**` and friends to be added as backport PRs against the stable branches, with the same reasoning as the current triggers (e.g. `build-images-ci.yaml` on `v1.10` has `push` triggers for `v1.10` instead of `master` as above).

Reply:
Yes, that was my intention :point_up:

Reply:
But don't we need to change the CI 3.0 workflows for stable branches here anyway? (They run the master versions.)

Reply:
I thought stable branches had their own version of the workflows targeting the stable branch itself :thinking: 

https://github.com/cilium/cilium/blob/1c4e32d7abf4306b7074af80a443e7032b6d8444/.github/workflows/build-images-ci.yaml#L10-L12

(or am I missing something?)

Reply:
I was referring to e.g. https://github.com/cilium/cilium/blob/master/.github/workflows/conformance-aks-v1.10.yaml but now I see we probably don't need to change anything as it's not the same kind of triggers.